src/outfit.ts

```diff
@@ -205,6 +213,28 @@ export function meatOutfit(embezzlerUp: boolean, requirement?: Requirement, sea?
       forceEquip.push($item`Kramco Sausage-o-Matic™`);
     }
   }
+
+  const stickerSlots = $slots`sticker1, sticker2, sticker3`;
+  const UPC = $item`scratch 'n' sniff UPC sticker`;
+
+  if (embezzlerUp) {
+    const currentWeapon = 25 * findLeprechaunMultiplier(meatFamiliar());
+    const embezzlers = globalOptions.ascending
+      ? 20
+      : Math.min(30, embezzlerCount() || digitizedMonstersRemaining());
```

Comment:
Should this not be `!globalOptions.ascending` instead, since we only get to keep the stickers for future garbo runs if we aren't ascending.
Also, why are we using a value of 30 again?